2023-11-03 15:27:58
2023-11-03 15:27:58 > docker-software-delivery@1.0.0 start /app
2023-11-03 15:27:58 > node ./bin/www
2023-11-03 15:27:58
2023-11-03 15:27:58 NODE_ENV: production
2023-11-03 15:27:58 /app/node_modules/mongodb/lib/mongo_client.js:421
2023-11-03 15:27:58           throw err
2023-11-03 15:27:58           ^
2023-11-03 15:27:58
2023-11-03 15:27:58 AssertionError [ERR_ASSERTION]: null == Error [MongoError]: failed to connect to server [item-db:27017] on first connect [Error: connect ECONNREFUSED 172.21.0.2:27017
2023-11-03 15:27:58     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1159:16) {
2023-11-03 15:27:58   name: 'MongoError'
2023-11-03 15:27:58 }]
2023-11-03 15:27:58     at Pool.<anonymous> (/app/node_modules/mongodb-core/lib/topologies/server.js:336:35)
2023-11-03 15:27:58     at Pool.emit (events.js:400:28)
2023-11-03 15:27:58     at Connection.<anonymous> (/app/node_modules/mongodb-core/lib/connection/pool.js:280:12)
2023-11-03 15:27:58     at Object.onceWrapper (events.js:520:26)
2023-11-03 15:27:58     at Connection.emit (e...
2023-11-03 15:27:58     at /app/app.js:24:10
2023-11-03 15:27:58     at connectCallback (/app/node_modules/mongodb/lib/mongo_client.js:527:5)
2023-11-03 15:27:58     at /app/node_modules/mongodb/lib/mongo_client.js:418:11
2023-11-03 15:27:58     at processTicksAndRejections (internal/process/task_queues.js:77:11) {
2023-11-03 15:27:58   generatedMessage: true,
2023-11-03 15:27:58   code: 'ERR_ASSERTION',
2023-11-03 15:27:58   actual: null,
2023-11-03 15:27:58   expected: Error [MongoError]: failed to connect to server [item-db:27017] on first connect [Error: connect ECONNREFUSED 172.21.0.2:27017
2023-11-03 15:27:58       at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1159:16) {
2023-11-03 15:27:58     name: 'MongoError'
2023-11-03 15:27:58   }]
2023-11-03 15:27:58       at Pool.<anonymous> (/app/node_modules/mongodb-core/lib/topologies/server.js:336:35)
2023-11-03 15:27:58       at Pool.emit (events.js:400:28)
2023-11-03 15:27:58       at Connection.<anonymous> (/app/node_modules/mongodb-core/lib/connection/pool.js:280:12)
2023-11-03 15:27:58       at Object.onceWrapper (events.js:520:26)
2023-11-03 15:27:58       at Connection.emit (events.js:400:28)
2023-11-03 15:27:58       at Socket.<anonymous> (/app/node_modules/mongodb-core/lib/connection/connection.js:189:49)
2023-11-03 15:27:58       at Object.onceWrapper (events.js:520:26)
2023-11-03 15:27:58       at Socket.emit (events.js:400:28)
2023-11-03 15:27:58       at emitErrorNT (internal/streams/destroy.js:106:8)
2023-11-03 15:27:58       at emitErrorCloseNT (internal/streams/destroy.js:74:3),
2023-11-03 15:27:58   operator: '=='
2023-11-03 15:27:58 }
2023-11-03 15:27:58 npm ERR! code ELIFECYCLE
2023-11-03 15:27:58 npm ERR! errno 1
2023-11-03 15:27:58 npm ERR! docker-software-delivery@1.0.0 start: `node ./bin/www`
2023-11-03 15:27:58 npm ERR! Exit status 1
2023-11-03 15:27:58 npm ERR!
2023-11-03 15:27:58 npm ERR! Failed at the docker-software-delivery@1.0.0 start script.
2023-11-03 15:27:58 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2023-11-03 15:27:58
2023-11-03 15:27:58 npm ERR! A complete log of this run can be found in:
2023-11-03 15:27:58 npm ERR!     /root/.npm/_logs/2023-11-03T08_27_58_819Z-debug.log
2023-11-03 15:28:00
2023-11-03 15:28:00 > docker-software-delivery@1.0.0 start /app
2023-11-03 15:28:00 > node ./bin/www
2023-11-03 15:28:00
2023-11-03 15:28:00 NODE_ENV: production
2023-11-03 15:28:00 Connected to database
2023-11-03 15:31:47 GET / 200 178.972 ms - 785
2023-11-03 15:31:47 GET /javascripts/dist/dist.min.js 200 4.300 ms - 186999
2023-11-03 15:31:47 GET /stylesheets/style.css 200 22.859 ms - 519
2023-11-03 15:31:47 GET /api 200 11.175 ms - 53
2023-11-03 15:27:57 2023-11-03T08:27:57.979+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=4c6ed5adb1fe
2023-11-03 15:27:57 2023-11-03T08:27:57.982+0000 I CONTROL  [initandlisten] db version v3.6.23
2023-11-03 15:27:57 2023-11-03T08:27:57.982+0000 I CONTROL  [initandlisten] git version: d352e6a4764659e0d0350ce77279de3c1f243e5c
2023-11-03 15:27:57 2023-11-03T08:27:57.982+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2023-11-03 15:27:57 2023-11-03T08:27:57.982+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2023-11-03 15:27:57 2023-11-03T08:27:57.982+0000 I CONTROL  [initandlisten] modules: none
2023-11-03 15:27:57 2023-11-03T08:27:57.982+0000 I CONTROL  [initandlisten] build environment:
2023-11-03 15:27:57 2023-11-03T08:27:57.982+0000 I CONTROL  [initandlisten]     distmod: ubuntu1604
2023-11-03 15:27:57 2023-11-03T08:27:57.982+0000 I CONTROL  [initandlisten]     distarch: x86_64
2023-11-03 15:27:57 2023-11-03T08:27:57.982+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2023-11-03 15:27:57 2023-11-03T08:27:57.982+0000 I CONTROL  [initandlisten] options: { net: { bindIpAll: true } }
2023-11-03 15:27:57 2023-11-03T08:27:57.986+0000 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2023-11-03 15:27:57 2023-11-03T08:27:57.986+0000 I STORAGE  [initandlisten]
2023-11-03 15:27:57 2023-11-03T08:27:57.986+0000 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2023-11-03 15:27:57 2023-11-03T08:27:57.986+0000 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2023-11-03 15:27:57 2023-11-03T08:27:57.986+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=3386M,cache_overflow=(file_max=0M),session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),compatibility=(release="3.0",require_max="3.0"),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2023-11-03 15:27:59 2023-11-03T08:27:59.186+0000 I STORAGE  [initandlisten] WiredTiger message [1699000079:186380][1:0x7fa4b56b8a40], txn-recover: Main recovery loop: starting at 2/12160
2023-11-03 15:27:59 2023-11-03T08:27:59.298+0000 I STORAGE  [initandlisten] WiredTiger message [1699000079:298919][1:0x7fa4b56b8a40], txn-recover: Recovering log 2 through 3
2023-11-03 15:27:59 2023-11-03T08:27:59.370+0000 I STORAGE  [initandlisten] WiredTiger message [1699000079:370463][1:0x7fa4b56b8a40], txn-recover: Recovering log 3 through 3
2023-11-03 15:27:59 2023-11-03T08:27:59.449+0000 I STORAGE  [initandlisten] WiredTiger message [1699000079:449259][1:0x7fa4b56b8a40], txn-recover: Set global recovery timestamp: 0
2023-11-03 15:27:59 2023-11-03T08:27:59.539+0000 I CONTROL  [initandlisten]
2023-11-03 15:27:59 2023-11-03T08:27:59.539+0000 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2023-11-03 15:27:59 2023-11-03T08:27:59.539+0000 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2023-11-03 15:27:59 2023-11-03T08:27:59.539+0000 I CONTROL  [initandlisten]
2023-11-03 15:27:59 2023-11-03T08:27:59.539+0000 I CONTROL  [initandlisten]
2023-11-03 15:27:59 2023-11-03T08:27:59.539+0000 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/enabled is 'always'.
2023-11-03 15:27:59 2023-11-03T08:27:59.539+0000 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2023-11-03 15:27:59 2023-11-03T08:27:59.539+0000 I CONTROL  [initandlisten]
2023-11-03 15:27:59 2023-11-03T08:27:59.567+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2023-11-03 15:27:59 2023-11-03T08:27:59.569+0000 I NETWORK  [initandlisten] listening via socket bound to 0.0.0.0
2023-11-03 15:27:59 2023-11-03T08:27:59.569+0000 I NETWORK  [initandlisten] listening via socket bound to /tmp/mongodb-27017.sock
2023-11-03 15:27:59 2023-11-03T08:27:59.569+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
2023-11-03 15:28:00 2023-11-03T08:28:00.400+0000 I NETWORK  [listener] connection accepted from 172.21.0.3:49988 #1 (1 connection now open)
2023-11-03 15:28:00 2023-11-03T08:28:00.408+0000 I NETWORK  [conn1] received client metadata from 172.21.0.3:49988 conn1: { driver: { name: "nodejs", version: "2.2.36" }, os: { type: "Linux", name: "linux", architecture: "x64", version: "5.15.90.1-microsoft-standard-WSL2" }, platform: "Node.js v14.21.3, LE, mongodb-core: 2.1.20" }
